src/main/java/uk/gov/companieshouse/acsp/models/filing/ServiceAddress.java

```diff
@@ -12,7 +12,7 @@ public final class ServiceAddress {
     @JsonProperty("address")
     private Address correspondenceAddress;
 
-    public boolean isServiceAddressROA() {
```

Comment:
Yes, As we are having the @JsonProperty("isServiceAddressROA") which is getting duplicated in response  as "isServiceAddressROA" & "serviceAddressROA". So solutions are either change the @JsonProperty(which we can't as it will mismatch in xml)  or change the getter name. So I chose later